### DISCUSSION TITLE
AsusFX504 - 0.9.7

### DISCUSSION THREAD

**Discussion by RobyRew** [RobyRew/ASUS-FX504GE-Hackintosh_OpenCore#21] [General]
### News / Fixes:
- Updated my personal Laptop to Sonoma 14.2.0
- Updated all config.plist parameters for OC 0.9.7, all OpenCore files and Kexts are also updated.
- Removed `enable-backlight-registers-fix` property from iGPU because now its no longer working, replaced with `enable-backlight-registers-alternative-fix` see more: https://github.com/acidanthera/WhateverGreen#intel-hd-graphics
- Small improvements and overall taken care of every detail of the EFI.

### Be aware:
- Right now this EFI is using [AirportItlwm-Sonoma-v2.3.0-DEBUG-alpha-ff1138b.zip](https://github.com/OpenIntelWireless/itlwm/releases/download/v2.3.0-alpha/AirportItlwm-Sonoma-v2.3.0-DEBUG-alpha-ff1138b.zip) the fact that its using Alpha software means that it might have some issues unresolved, but nothing "Stable" is provided for Sonoma yet.

### Known Issues:
- HDMI is not working on this Release, it might be that this fix (`agdpmod | Data | 76697439363936` [source reddit: ](https://www.reddit.com/r/hackintosh/comments/10ywrx6/comment/j804eqb/) is no longer working, i need help again).

_Pull Requests are welcome! :)_
Don't Forget to Star the Repository ⭐️

<hr /><em>This discussion was created from the release <a href='https://github.com/RobyRew/ASUS-FX504GE-Hackintosh_OpenCore/releases/tag/0.9.7'>AsusFX504 - 0.9.7</a>.</em>

**Comment by Happsz1** [comment]
Really a great release and was worth updating to Sonoma! 

Not sure about HDMI "not working" being listed though cuz in my case it works (but then again, it could be just me modifying the config file and adding "enable-hdmi20" or the hardware is different enough to have a different result lol)
![IMG_3396](https://github.com/RobyRew/ASUS-FX504GE-Hackintosh_OpenCore/assets/131194487/9008d7cb-2d4c-4338-be9f-afb7e3c08ca6)

**Reply by Dv1101** [reply]
can u share modified config file? i want to use my monitor :)

**Reply by Happsz1** [reply]
Oh, here's the modified config file lol 
https://drive.google.com/file/d/1Cn8o4gyQJbAYNSp8PlaBzOclWDGogJac/view?usp=sharing
(had to use gdrive cuz GitHub here won't let me attach the file here lol)

Although to be honest, this modified config is so extremely simple 😂

(not sure if this all works with all monitors cuz my laptop does have different behaviors every time I use a different monitor such as my TCL TV being flawless while my drawing tablet having screen flickering sometimes)

**Comment by Fauzanmhr** [comment]
first i want to thank you and i have question : i try this EFI on Sonoma its working but why when i try on Ventura, my fx504ge but boot loop between apple logo and ASUS logo?
